search/tests/test_search_result_processor.py

```diff
@@ -25,8 +25,8 @@ def test_strings_in_dictionary(self):
         })
         get_strings = SearchResultProcessor.strings_in_dictionary(test_dict)
         self.assertEqual(len(get_strings), 2)
-        self.assertEqual(get_strings[0], test_dict["a"])
```

Comment:
It seems like the list indexes does not remain in the same order so it was causing flakiness. I think all we need to check is that the strings are in the list and order of them does not matter. Correct me if I wrong.